src/quest/extras/sql.py

```diff
@@ -0,0 +1,88 @@
+from .. import WorkflowFactory, WorkflowManager, PersistentHistory, History, BlobStorage, Blob
+
+try:
+    from sqlalchemy import create_engine, Column, Integer, String, JSON, Engine
+    from sqlalchemy.orm import sessionmaker, Session, declarative_base
+except ImportError:
+    raise ImportError("The 'sql' extra is required to use this module. Run 'pip install quest-py[sql]'.")
+
+Base = declarative_base()
+
+
+class RecordModel(Base):
+    __tablename__ = 'records'
+
+    id = Column(Integer, primary_key=True, autoincrement=True)
+    name = Column(String)  # TODO good name for this?
+    key = Column(String)
+    blob = Column(JSON)
+
+    def __repr__(self):
+        return f'<{self.__class__.__name__}: {self.name}>'
```

Comment:
Is this standard? I saw some code that did this and was wondering if we need this?